compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformTextInputMethodRequest.skiko.kt

```diff
@@ -38,4 +39,6 @@ actual interface PlatformTextInputMethodRequest {
     val editProcessor: EditProcessor?
     @ExperimentalComposeUiApi
     val textLayoutResult: Flow<TextLayoutResult>
+    @ExperimentalComposeUiApi
```

Comment:
Can we start writing KDocs here? It's a public API (I know it's only for cross-module things, but anyway), and we need to document it.

Reply:
Added KDocs.